charts/Makefile

```diff
@@ -2,35 +2,63 @@ OCP_VERSION ?= 4.19
 BRANCH ?= master
 SYNC2CHARTS ?= true
 ORGREPO ?= https://github.com/openshift
+PROJECTS = \
+	cluster-api \
+	cluster-api-provider-aws \
+	cluster-api-provider-openshift-assisted
 
-build-cluster-api-chart: PROJECT = cluster-api
-build-cluster-api-aws-chart: PROJECT = cluster-api-provider-aws
-build-cluster-api-azure-chart: PROJECT = cluster-api-provider-azure
-
+ifeq ($(BRANCH),master)
+	TAG := latest
+else
+	TAG := $(patsubst release-%, %,$(BRANCH))
+endif
+CHART_VERSION = $(OCP_VERSION)
+CHART_APP_VERSION = $(OCP_VERSION)
+CHART_VALUES_IMAGE_TAG = $(OCP_VERSION)
+CHART_VALUES_IMAGE_TAG_PREFIX = "v"
+# Define specific overrides
+ORGREPO_cluster-api-provider-openshift-assisted = https://github.com/openshift-assisted
+KUSTOMIZE_CONFIG_DIRS_cluster-api-provider-openshift-assisted = "bootstrap" "controlplane"
+CHART_VERSION_cluster-api-provider-openshift-assisted = "1.0.0"
```

Comment:
I've hardcoded to 1.0.0 while we decide what would be the best strategy to version our chart version

Reply:
gotcha. AFAIU 1 z release of MCE will always have 1 and only 1 version for each chart, right? It might or might not change between x, y and z but never more than one. Is this correct? If so, we could probably set the chart version as MCE x.y.z